.github/workflows/spread.yaml

```diff
@@ -0,0 +1,46 @@
+name: Spread
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: "0 0 */2 * *"
+
+jobs:
+  snap-build:
+    runs-on: ubuntu-20.04
```

Comment:
Why not jammy?

Reply:
I would rather not, this is a GHA provided instance and 20.04 is stable; this is mostly a platform for us to run snapcraft which creates a lxd container with the appropriate target for the base.